modules/examiner/php/addexaminer.class.inc

```diff
@@ -64,6 +64,21 @@ class AddExaminer extends \NDB_Page
         $fullName = $values['addName'] ?? '';
         $siteID   = $values['addSite'] ?? '';
 
+        // check for site permissions
+        $user = \User::singleton();
+        if (!$user->hasAnyPermission(
+            [
+                'superuser',
```

Comment:
superuser should not be here. (hasAnyPermission is a wrapper around hasPermission which already handles it.)